### ISSUE TITLE
Charset will be lost in the content type of the response header of the HTTP request

### ISSUE THREAD

**Issue by wangfan0524** [prometheus/prometheus#9930]
When querying the Prometheus data through the HTTP request, the content type in the response header finally loses the charset. Why？

![image](https://user-images.githubusercontent.com/37073064/144528541-b9391310-53be-4f40-86aa-b34b95dc1c2b.png)

**Comment by roidelapluie** [comment]
it seems that we could always return UTF-8, but I do not know how breaking that change would be.

@juliusv what is your opinion?

**Comment by juliusv** [comment]
So we just always set `Content-type: application/json` without charset here: https://github.com/prometheus/prometheus/blob/46529bf8511470141a19cd774e3d14562758483f/web/api/v1/api.go#L1495

And according to [RFC4627](https://www.ietf.org/rfc/rfc4627.txt), JSON defaults to UTF-8: "JSON text SHALL be encoded in Unicode. The default encoding is UTF-8.". And https://security.stackexchange.com/questions/82913/benefit-to-specifying-json-content-type-character-set sounds like there's no real benefit to adding the charset for JSON.

The screenshot above makes it look like @wangfan0524 assumed that the `Content-Type` header from the request would be passed through to the response? Indeed, that is not what the `Content-Type` in the request is about - it only specifies the `Content-Type` of the request body itself and has no influence on the response (in contrast to the `Accept` and `Accept-Charset` etc. headers).

In short, I don't think there's anything we need to do here.

**Comment by roidelapluie** [comment]
Thanks Julius.  Based on your detailed input, I will close this issue.
